chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
I come to think `None` is better than `-1` for indicating invalid `previous_epoch_detail` because if someone calculates some values from invalid `previous_epoch_detail` with some mathematical operations, it will raise error if we use `None`, while return an invalid value without error if `-1`. So, `None` is safer, I think.